01_01_Basic_Rules.txt

```diff
@@ -765,16 +767,14 @@ standard Hip Firing rules apply.
 Making Melee Attacks:
 	Melee combat is relatively simple. To hit an enemy player: add your 
 dexterity combat modifier to a d10 roll plus your combat modifier from your 
-weapon/attack*, and add any relevant skill(s)**. The result must be greater than 
-the opponent's Guard DC: 
+weapon/attack*. The result must be greater than the opponent's Guard DC: 
 
-	(DEX * 4)/10 + any skill** in combat. 
+	(DEX * 4)/10  

```

Comment:
... what was the point of that in the first place?

Reply:
I honestly have no idea :(

Reply:
The point in "any skill** in combat. "?

Reply:
Melee Combat Skill used to contribute to melee accuracy

Reply:
When skill was being added to attacks, adding it to defense too was the only way to maintain any balance. Removing the one necessitates the removal of the other.